src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
 
 # Documenting Unison Code
 
-Unison documentation is written in Unison. Documentation consists of values of the type `builtin.Doc`, and there's a special syntax to create these values:
+Unison ships with a powerful documentation format. Capable of expressing far more than markdown or static text, the documentation format empowers developers to write rich, clear, and correct docs. Documentation is a first class feature of the Unison language and is captured by the Unison type: `builtin.Doc2`. Unison docs offer native support for features like executing typechecked code samples, embedding docs within other docs, and annotating your writing with tooltips, asides, and callouts. We'll walk through some of these features next. 
 
-```unison
-myDoc : Doc
-myDoc = [: This is some documentation. :]
-```
-
-The syntax `[:` opens a documentation block, and `:]` closes it. Everything in between is documentation. The block can even span multiple lines:
+## The basics 
+Documentation blocks start with `{{` and end with a matching `}}`. Anonymous documentation blocks can be defined immediately before a top-level term or type but you can also create them anywhere you'd write an expression. 
 
 ```unison
-myDoc = [:
-  This is documentation
-  with line breaks and indentation.
-:]
+author = {{Noam Chomsky}}
+{{@eval{nonsense} - *{{author}}*, is a grammatically well formed, but non-sensical sentence}}
+nonsense = "Colorless green ideas sleep furiously."
```

Comment:
I found this example difficult to follow. It's not immediately clear what's meta-level or meaningful. Maybe it's just that this sentence is about sentences. It's clear upon reflection, but it would be nice to avoid reflection.

It adds to the confusion that `nonsense` is a string. An obvious question is "why evaluate this?". Maybe have it evaluate an arithmetic expression instead.